app/views/layouts/application.html.erb

```diff
@@ -55,7 +74,7 @@
         <div class="footer_info">
           <p>Created by the <a href="http://cdrh.unl.edu">Center for Digital Research in the Humanities</a>.</p>
           <div class="footer_logo">
-            <a href="http://www.unl.edu"><%# image_tag 'unl_black.png' %></a>
+            <a href="http://www.unl.edu"><%= image_tag 'unl_black.png' %></a>
```

Comment:
Will leave this in and add the favicon from the other branch after this is merged